en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,77 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signin) or use your Github account if you have one (see Github instructions below).
```

Comment:
"GitHub" has a capital "H" and as a brand name should be spelled always the same.
```suggestion
2. Sign up for an account (https://glitch.com/signin) or use your GitHub account if you have one (see GitHub instructions below).
```